rotkehlchen/externalapis/opensea.py

```diff
@@ -57,6 +93,7 @@ def serialize(self) -> dict[str, Any]:
             'description': self.description,
             'large_image': self.large_image,
             'floor_price': str(self.floor_price) if self.floor_price else None,
+            'floor_price_asset': self.floor_price_asset.identifier if self.floor_price_asset else None,  # noqa: E501
```

Comment:
When can this be None? You initialize it as A_ETH and the type is `Asset`, not `Asset | None`

Reply:
i forgot to change it when doing changes. I've removed it